app/src/main/res/values/strings.xml

```diff
@@ -1273,7 +1273,7 @@
         <item><![CDATA[<a href=https://github.com/cao->Matteo Caoduro</a>]]></item>
         <item><![CDATA[<a href=https://github.com/Verdulo>Verdulo</a>]]></item>
         <item><![CDATA[<a href=https://github.com/Raulvo>Raulvo</a>]]></item>
-        <item><![CDATA[<a href=https://github.com/Raulvo>Raulvo</a>]]></item>
+        <item><![CDATA[<a href=https://github.com/Raulvo>Raulvo</a>|<a href=https://github.com/gnusuari0>gnusuari0</a>]]></item>
```

Comment:
The attribution can be whatever you want, link or no link, your choice. This gets displayed in the about dialog, readme, changelog, etc.

Reply:
Ok, thanks. The link can be left there.